petri/src/vm/openvmm/modify.rs

```diff
@@ -136,6 +136,21 @@ impl PetriVmConfigOpenVmm {
         self
     }
 
+    /// Enable TPM state persistence
+    pub fn with_tpm_state_persistence(mut self) -> Self {
+        if !self.firmware.is_openhcl() {
+            panic!("TPM state persistence is only supported for OpenHCL.")
```

Comment:
Are these changes not getting backported?

Reply:
OpenHCL here is to distinguish with OpenVMM-based tests, not with legacy HCL.